src/firebolt/common/base_cursor.py

```diff
@@ -243,6 +248,17 @@ def _reset(self) -> None:
         self._next_set_idx = 0
         self._query_id = ""
 
+    def _parse_response_headers(self, headers: Headers) -> None:
+        """Parse response and update relevant cursor fields."""
+        update_parameters = headers.get("Firebolt-Update-Parameters")
+        # parse update parameters dict and set keys as attributes
+        if update_parameters:
+            # parse key1=value1,key2=value2 comma separated string into dict
+            param_dict = dict(item.split("=") for item in update_parameters.split(","))
```

Comment:
Just a suggestion, maybe we can log with a debug level if received parameter is not known. Might help with debugging in some cases

Reply:
Fair point.